### PR TITLE
[test](inverted index)Add upgrade and downgrade test cases for inverted_index

### PR DESCRIPTION
**Pull request by ChenPeng2013**
## Proposed changes

Issue Number: close #xxx

add case

### CODE REVIEW

File: regression-test/suites/inverted_index_up_down_p0/load.groovy [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+suite("test_upgrade_downgrade_prepare_inverted_index","p0,inverted_index,restart_fe") {
+    for (version in ["V1", "V2"]) {
+        sql "drop table if exists t_up_down_inverted_index${version}_duplicate"
+        sql String.format('''
+        create table t_up_down_inverted_index%s_duplicate(
+        a int,
+        b int,
+        c int,
+        index idx_a(a)
```
Comment (qidaye) [inline regression-test/suites/inverted_index_up_down_p0/load.groovy:26]: inverted_index should contain fulltext with tokenized, fulltext without tokenized and numerical value.

File: regression-test/suites/inverted_index_up_down_p0/inverted_index.groovy [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        def tableName = "t_up_down_inverted_index${version}_duplicate"
+        def result = sql(String.format("show create table %s", tableName))
+        if (!result[0][1].toString().contains(String.format('''"inverted_index_storage_format" = "%s"''', version))) {
+            throw new IllegalStateException("${tableName} inverted_index_storage_format exception")
+        }
+        result = sql("select count(*) from ${tableName} where a = 10")
+        if (result[0][0] != 10) {
+            throw new IllegalStateException("count error")
+        }
+        sql "alter table ${tableName} add index idx_b(b)"
```
Comment (qidaye) [inline regression-test/suites/inverted_index_up_down_p0/inverted_index.groovy:29]: Run `build index` after create a new index.
And check indexes count after build.

**Comment by hello-stephen** [comment]
run buildall

**Review by qidaye — COMMENTED** [review]
Add session variable before execute query.
`sql "SET enable_match_without_inverted_index = false"`

**Review by qidaye — APPROVED** [review]
LGTM

**Comment by qidaye** [comment]
run build'all

**Comment by qidaye** [comment]
run buildall

**Comment by ChenPeng2013** [comment]
run buildall

**Comment by hello-stephen** [comment]
skip buildall

**Pull request merged**